<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>901</width>
    <height>586</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>686</width>
    <height>434</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>DOF Calculator</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="spacing">
     <number>10</number>
    </property>
    <item row="1" column="1">
     <widget class="QTableWidget" name="table_dof">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>280</width>
        <height>86</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>280</width>
        <height>86</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::HLine</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="autoScroll">
       <bool>false</bool>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::NoSelection</enum>
      </property>
      <property name="rowCount">
       <number>4</number>
      </property>
      <property name="columnCount">
       <number>1</number>
      </property>
      <attribute name="horizontalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderDefaultSectionSize">
       <number>21</number>
      </attribute>
      <attribute name="verticalHeaderHighlightSections">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderMinimumSectionSize">
       <number>21</number>
      </attribute>
      <attribute name="verticalHeaderStretchLastSection">
       <bool>false</bool>
      </attribute>
      <row>
       <property name="text">
        <string>Глубина резкости</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>Передний фронт ГРИП</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>Задний фронт ГРИП</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>Гиперфокальное расст</string>
       </property>
      </row>
      <column/>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QFrame" name="frame_control">
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <widget class="QSlider" name="slider_focallen">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>378</y>
         <width>140</width>
         <height>19</height>
        </rect>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>1000</number>
       </property>
       <property name="singleStep">
        <number>10</number>
       </property>
       <property name="pageStep">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="invertedAppearance">
        <bool>false</bool>
       </property>
       <property name="invertedControls">
        <bool>false</bool>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TicksBelow</enum>
       </property>
       <property name="tickInterval">
        <number>70</number>
       </property>
      </widget>
      <widget class="QSlider" name="slider_distance">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>258</y>
         <width>140</width>
         <height>19</height>
        </rect>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>50</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::NoTicks</enum>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="spin_distance">
       <property name="geometry">
        <rect>
         <x>189</x>
         <y>250</y>
         <width>81</width>
         <height>22</height>
        </rect>
       </property>
       <property name="suffix">
        <string> м</string>
       </property>
       <property name="decimals">
        <number>1</number>
       </property>
       <property name="minimum">
        <double>1.000000000000000</double>
       </property>
       <property name="maximum">
        <double>50.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.500000000000000</double>
       </property>
      </widget>
      <widget class="QLabel" name="label_background">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>8</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Задний план</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_crop">
       <property name="geometry">
        <rect>
         <x>161</x>
         <y>173</y>
         <width>70</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>Кроп-фактор</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_focallen">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>350</y>
         <width>241</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Эквивалентное фокусное расстояние </string>
       </property>
      </widget>
      <widget class="QLabel" name="label_aperture">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>400</y>
         <width>171</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Диафрагмагменное число</string>
       </property>
      </widget>
      <widget class="QSlider" name="slider_aperture">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>430</y>
         <width>171</width>
         <height>19</height>
        </rect>
       </property>
       <property name="toolTipDuration">
        <number>-1</number>
       </property>
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>32</number>
       </property>
       <property name="pageStep">
        <number>0</number>
       </property>
       <property name="value">
        <number>1</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TicksBelow</enum>
       </property>
      </widget>
      <widget class="QComboBox" name="combo_model">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>78</y>
         <width>220</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="spin_crop">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>191</y>
         <width>70</width>
         <height>22</height>
        </rect>
       </property>
       <property name="minimum">
        <double>0.100000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="spin_focallen">
       <property name="geometry">
        <rect>
         <x>189</x>
         <y>380</y>
         <width>81</width>
         <height>22</height>
        </rect>
       </property>
       <property name="suffix">
        <string> мм</string>
       </property>
       <property name="decimals">
        <number>1</number>
       </property>
       <property name="minimum">
        <double>1.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1000.000000000000000</double>
       </property>
      </widget>
      <widget class="QLabel" name="label_model">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>58</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Модель</string>
       </property>
      </widget>
      <widget class="QComboBox" name="combo_sensor">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>191</y>
         <width>140</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_sensor">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>171</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Формат кадра</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_distance">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>238</y>
         <width>141</width>
         <height>10</height>
        </rect>
       </property>
       <property name="text">
        <string>Расстояние до модели</string>
       </property>
      </widget>
      <widget class="QComboBox" name="combo_background">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>28</y>
         <width>220</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QSlider" name="slider_backgroun_distance">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>310</y>
         <width>141</width>
         <height>22</height>
        </rect>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>500</number>
       </property>
       <property name="singleStep">
        <number>1</number>
       </property>
       <property name="value">
        <number>1</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="spin_backgrond_distance">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>310</y>
         <width>81</width>
         <height>22</height>
        </rect>
       </property>
       <property name="suffix">
        <string> м</string>
       </property>
       <property name="decimals">
        <number>2</number>
       </property>
       <property name="minimum">
        <double>1.000000000000000</double>
       </property>
       <property name="maximum">
        <double>500.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>1.000000000000000</double>
       </property>
       <property name="value">
        <double>1.000000000000000</double>
       </property>
      </widget>
      <widget class="QLabel" name="label_background_meters">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>290</y>
         <width>181</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Расстояние до фона</string>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="spin_aperture">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>430</y>
         <width>62</width>
         <height>22</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="decimals">
        <number>1</number>
       </property>
       <property name="minimum">
        <double>1.000000000000000</double>
       </property>
       <property name="maximum">
        <double>32.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.400000000000000</double>
       </property>
      </widget>
      <widget class="QLabel" name="diafragm_letter">
       <property name="geometry">
        <rect>
         <x>198</x>
         <y>433</y>
         <width>31</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>f/</string>
       </property>
      </widget>
      <widget class="QComboBox" name="combo_strategy">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>130</y>
         <width>220</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_strategy">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>110</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Режим съемки</string>
       </property>
      </widget>
     </widget>
    </item>
    <item row="0" column="0" rowspan="2">
     <widget class="QLabel" name="image">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>combo_background</tabstop>
  <tabstop>combo_model</tabstop>
  <tabstop>combo_sensor</tabstop>
  <tabstop>spin_crop</tabstop>
  <tabstop>slider_distance</tabstop>
  <tabstop>spin_distance</tabstop>
  <tabstop>slider_focallen</tabstop>
  <tabstop>spin_focallen</tabstop>
  <tabstop>slider_aperture</tabstop>
  <tabstop>table_dof</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
